CHANGELOG.md

```diff
@@ -95,6 +95,9 @@ that have many transactions.
   entire set of results would need to be queried every time to discover
   which coins have been spent and are currently available.
 - `GET /` has new fields `.indexes.{addr,tx}` for the status of indexers.
+- `GET /fee?blocks=nBlocks` is now accesible from the terminal command `fee`
```

Comment:
So this section is about the API not about the CLI or the client, so I don't think it's really worth it to mention changes to the CLI here as that could be confusing. The API here hasn't changed, so this change can be omitted.

Reply:
The change that should be noted however is for the client.

Reply:
Cool, updated CHANGELOG and also (while there) added a few other details we missed in previous PRs.